02c_layout.ipynb

```diff
@@ -0,0 +1,752 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "35b6b8f5-acd9-4ad8-8987-f1d7050a8061",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#| export\n",
+    "# %answer key/dashboard/main.py 31\n",
+    "\n",
+    "selected_data_accordion.selected_index = 0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee491de4-e373-4460-b576-4b4d323b6fad",
+   "metadata": {},
+   "source": [
+    "We like this trick, because our users are probably more interested in our plot than the raw data anyways, so this keeps the raw data from being in the way."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4ccc1ed5-c59c-4853-a959-6a9160d5a4c2",
+   "metadata": {},
+   "source": [
+    "### Add Descriptive Text with Links\n",
+    "\n",
+    "Before we put everything together, let's add some widgets that provide information about the dashboard. Because we want to take advantage of hyperlinking (to properly cite our sources), we will use the `HTML` widget. The `HTML` widget allows you to display text that can be styled using HTML and CSS.  First let's export the HTML we will want to display as two constants."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ee3f3e71-2120-4415-81e1-ff9f36ef6204",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#| export\n",
+    "INTRO_TEXT = '''\n",
+    "<p><b>Curve Smoothing</b>\n",
+    "This tool is for smoothing and selecting global mean surface temperature data for visualization. Start by selecting a date\n",
+    "range, and then select the smoothing parameters you want to use. Then click through to the next step, where you will change properties\n",
+    "of the curve smoothing algorithm you selected and visualize the data.\n",
+    "</p>\n",
+    "'''\n",
+    "SOURCES_TEXT = '''\n",
+    "<p>\n",
+    "<b>About Global Mean Surface Temperature Data</b>\n",
+    "<a href=\"https://climate.nasa.gov/vital-signs/global-temperature/\"\n",
+    "target=\"_blank\">Global Temperature (NASA)</a>,\n",
+    "<a href=\"https://data.giss.nasa.gov/gistemp/graphs/\"\n",
+    "target=\"_blank\">GISS Surface Temperature Analysis (NASA)</a>\n",
+    "</p><p>\n",
+    "This site is based on data downloaded from the following site on 2024-06-17:\n",
+    "<a href=\"https://data.giss.nasa.gov/gistemp/graphs/graph_data/Global_Mean_Estimates_based_on_Land_and_Ocean_Data/graph.csv\"\n",
+    "target=\"_blank\">Global Annual Mean Surface Air Temperature Change (NASA)</a>\n",
+    "'''"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e9d46022-ef0a-4d0b-bb46-033cd9f8330b",
+   "metadata": {},
+   "source": [
+    "### Layout widget objects\n",
```

Comment:
I don't think we need to point this out, but the `Layout` is itself a widget, so you can observe changes in its properties if you want.